### DISCUSSION TITLE
Online Internet Browser is here!

### DISCUSSION THREAD

**Discussion by VendelinSlezak** [VendelinSlezak/BleskOS#15] [Announcements]
I've finished first working version.

BleskOS now has simple internet stack which work with HTTP pages, however I added workaround that allowed to browse also HTTPS pages as HTTP, so BleskOS should open almost every website.

![VirtualBox_BleskOS_31_03_2022_19_24_00](https://user-images.githubusercontent.com/43180618/161115917-ed2b970a-1b1d-4aa6-9873-431becc77774.png)

![VirtualBox_BleskOS_31_03_2022_19_24_44](https://user-images.githubusercontent.com/43180618/161115940-d5e41b6b-1277-4126-9d78-5df40af8947c.png)

![VirtualBox_BleskOS_31_03_2022_19_26_18](https://user-images.githubusercontent.com/43180618/161115952-112b5dfe-4599-4ca3-9497-fbbf0bdceb24.png)

As you can see, rendering is on very simple level now, but it is enough to read pages on some basic level.

I hope you will enjoy it 😁

Stay tuned for next updates 😉

**Comment by Aap-Dev** [comment]
This is amazing! Are you planning on creating a javascript engine aswell?

**Comment by VendelinSlezak** [comment]
Yes, support for javascript is on my list, but before javascript comes CSS, and before CSS I want to take some break from internet things to work on some other important features like USB 3 or PNG or scalable font.

However, I am now working on another great program, table editor, what is something like office program Excel. I’ve already wrote half of it few weeks ago, so now I am finishing it to usable version. I hope I will release it next week.
